Userland/Utilities/tar.cpp

```diff
@@ -102,8 +102,17 @@ int main(int argc, char** argv)
                     }
                     break;
                 }
+                case Archive::TarFileType::GlobalExtendedHeader: {
+                    warnln("ignoring global extended header: {}", header.filename());
+                    break;
+                }
+                case Archive::TarFileType::ExtendedHeader: {
+                    warnln("ignoring extended header: {}", header.filename());
```

Comment:
I don't think the user needs to know about this?
`dbgln` would likely suffice.

Reply:
`dbgln` looks perfect for this, thanks!

I amended and force-pushed because the CI informed me that my commit message lacked a subject.